src/frontend/src/btc/stores/btc-transactions.store.ts

```diff
@@ -0,0 +1,41 @@
+import type { BtcTransactionUi } from '$btc/types/btc';
+import { initCertifiedStore, type CertifiedStore } from '$lib/stores/certified.store';
+import type { CertifiedData } from '$lib/types/store';
+import type { TokenId } from '$lib/types/token';
+import { nonNullish } from '@dfinity/utils';
+
+export type BtcCertifiedTransaction = CertifiedData<BtcTransactionUi>;
+
+export type BtcTransactionsData = BtcCertifiedTransaction[];
+
+export interface BtcTransactionsStore extends CertifiedStore<BtcTransactionsData> {
+	prepend: (params: { tokenId: TokenId; transactions: BtcCertifiedTransaction[] }) => void;
+}
+
+const initBtcTransactionsStore = (): BtcTransactionsStore => {
+	const { subscribe, update, reset } = initCertifiedStore<BtcTransactionsData>();
+
+	return {
+		prepend: ({
```

Comment:
Is it the exact same code as IC transactions stores?

Reply:
With a small adjustment to BtcTransactionUi type (in another PR), the stores are now 100% identical. I extracted it in a separate store in `lib`. BTC already uses it in this PR. And for IC, I'll refactor the code separately after we merge this feature branch to main.